File: csharp/ToolGood.Algorithm/Internals/ParameterType.cs
using System;
using System.Collections.Generic;
using System.Text;
using ToolGood.Algorithm.Enums;

namespace ToolGood.Algorithm.Internals
{
	/// <summary>
	/// 参数类型类
	/// </summary>
	public class ParameterType
	{
		/// <summary>
		/// 参数名称
		/// </summary>
		public string Name { get; internal set; }
		/// <summary>
		/// 参数类型
		/// </summary>
		public OperandType Type { get; internal set; }

		/// <summary>
		/// 操作符，可为空
		/// </summary>
		public string Operator { get; internal set; }

		/// <summary>
		/// 操作值，可为空
		/// </summary>
		public string Value { get; internal set; }
	}
}

File: java/src/main/java/toolgood/algorithm/internals/ParameterType.java
/**
 * 参数类型类
 */
package toolgood.algorithm.internals;

import toolgood.algorithm.enums.OperandType;

public class ParameterType {
    /**
     * 参数名称
     */
    private String name;
    /**
     * 参数类型
     */
    private OperandType type;

    /**
     * 操作符，可为空
     */
    private String operator;

    /**
     * 操作值，可为空
     */
    private String value;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public OperandType getType() {
        return type;
    }

    public void setType(OperandType type) {
        this.type = type;
    }

    public String getOperator() {
        return operator;
    }

    public void setOperator(String operator) {
        this.operator = operator;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }
}
